completion/functions/compstyle_grmlin_setup

```diff
@@ -0,0 +1,189 @@
+#!/bin/zsh
+
+### grml Project information:
+## Home:    https://grml.org
+## Authors: grml-team (grml.org)
+## License: GPL v2
```

Comment:
As you mentioned in your comment, you weren't sure about this - unfortunately, the terms of the GPL don't allow including GPL licensed code in MIT licensed projects like this, so I think this one should be removed. It's a bit of a grey area because it's "just" configuration, but I think it would be easiest to drop this one for now.

Reply:
It would be easy enough to include as its own separate GPL'ed plugin for grml fans. I'll drop it.

Reply:
Yep - maybe we can include something later, I just don't want to think about the potential licensing headache just yet.